cmd/dmverity-vhd/main.go

```diff
@@ -431,8 +432,8 @@ var createVHDCommand = cli.Command{
 	Usage: "creates LCOW layer VHDs inside the output directory with dm-verity super block and merkle tree appended at the end",
 	Flags: []cli.Flag{
 		cli.StringFlag{
-			Name:     imageFlag + ",i",
-			Usage:    "Required: container image reference",
+			Name:     inputFlag + ",i",
```

Comment:
mior nit: we may want to keep the old flag name around just to avoid breaking current usage
```suggestion
			Name:     inputFlag + ",image,i",
```